examples/echoserver/README_wolfsentry.md

```diff
@@ -0,0 +1,276 @@
+# wolfSentry Itegration Example
+
+This is a basic example that uses wolfsentry with SSH server. wolfSentry will filter a ssh client when it types invalid password more than a threshold value.
+Once the peer breaks the theshold number, the peer is accommodated into a penalty box with the certain penalty time.
+
+# Prerequisites
+
+This example is designed to be run an environment that has file system because JSON configuration file is expected to be read.
```

Comment:
"designed to be run an" should be "designed to be run in an".  "because JSON configuration file" should be "because a JSON configuration file"

Reply:
thanks. fixed